src/utils/interpolation.jl

```diff
@@ -1,3 +1,9 @@
+
+
+getrange(t, n) = LinRange(minimum(t), maximum(t), n)
```

Comment:
```suggestion
getrange(t, n) = range(minimum(t); stop = maximum(t), length = n)
```
It is recommended to use the `range` function instead of direct constructors of ranges.